### PR TITLE
add GLenums to getVertexAttrib in conformance test for WebGL 2

### PR DESCRIPTION
**Pull request by Richard-Yunchao**
PTAL when you have time. Thanks a lot!

### CODE REVIEW

File: sdk/tests/conformance/resources/gl-object-get-calls.js [hunk 1]
```diff
@@ -420,12 +435,11 @@ var validArrayForVertexAttrib = new Array(
     gl.VERTEX_ATTRIB_ARRAY_STRIDE,
     gl.VERTEX_ATTRIB_ARRAY_TYPE,
     gl.VERTEX_ATTRIB_ARRAY_NORMALIZED,
-    gl.VERTEX_ATTRIB_ARRAY_STRIDE,
-    gl.VERTEX_ATTRIB_ARRAY_ENABLED,
```
Comment (kenrussell) [inline sdk/tests/conformance/resources/gl-object-get-calls.js]: Why were the invalid argument tests for these two enums removed? This change does not look good.

**Comment by kenrussell** [comment]
The overall addition looks fine but the removal of existing tests does not.

**Comment by Richard-Yunchao** [comment]
@kenrussell , the removed GLenums are redundant. They have been included in the array.

**Comment by kenrussell** [comment]
Ah, sorry, I missed that. Thanks for this contribution then.

**Pull request merged**